js/fullscreen.js

```diff
@@ -0,0 +1,73 @@
+
+const body = document.querySelector('body');
+const fullscreenPhotoModal = document.querySelector('.big-picture');
+const socialCommentCount = document.querySelector('.social__comment-count');
+const commentsLoader = document.querySelector('.comments-loader');
+
+const likesCount = document.querySelector('.likes-count');
+const commentsCount = document.querySelector('.comments-count');
+const commentsList = document.querySelector('.social__comments');
+const photoCaption = document.querySelector('.social__caption');
+const fullscreenPhotoImage = fullscreenPhotoModal.querySelector('.big-picture__img img');
+const closeButton = document.querySelector('#picture-cancel');
+
+const AVATAR_WIDTH = 35;
+const AVATAR_HEIGHT = 35;
+
+
+function createFullscreenPhoto(url, description, likes, comments) {
+  fullscreenPhotoModal.classList.remove('hidden');
+  socialCommentCount.classList.add('hidden');
+  commentsLoader.classList.add('hidden');
+  body.classList.add('modal-open');
+
+  fullscreenPhotoImage.src = url;
+  photoCaption.textContent = description;
+  likesCount.textContent = likes;
+  commentsCount.textContent = comments.length;
+  createComments(comments);
+}
+
+function createComments(comments) {
+  commentsList.innerHTML = '';
+
+  comments.forEach(({avatar, name, message}) => {
+
+    const newCommentItem = document.createElement('li');
+    const newAvatar = document.createElement('img');
+    const newCommentText = document.createElement('p');
```

Comment:
Элементы выбранные через querySelector - должны заканчиваться на Element
```suggestion
    const newCommentItemElement = document.createElement('li');
    const newAvatarElement = document.createElement('img');
    const newCommentTextElement = document.createElement('p');
```